eventkit_cloud/settings/prod.py

```diff
@@ -202,3 +202,5 @@
 }
 
 DISABLE_SSL_VERIFICATION = os.environ.get('DISABLE_SSL_VERIFICATION', False)
+
+MAX_EXPORTRUN_EXPIRATION_DAYS = 45
```

Comment:
Additionally this setting should be added to the config endpoint, so that the UI can provide information, you can append it here. https://github.com/venicegeo/eventkit-cloud/blob/master/eventkit_cloud/ui/views.py#L201